data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,851 @@
+
+
+outfit "SC-12 Plasma Thruster"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 98000
+	thumbnail "outfit/small plasma thruster scin"
+	"mass" 25
+	"outfit space" -25
+	"engine capacity" -25
+	"thrust" 10.6
+	"thrusting energy" 0.8
+	"thrusting heat" 2.9
+	"flare sprite" "effect/scin flare/small"
+		"frame rate" 0.5
+	"flare sound" "scplasma small"
+	description `Rather than developing on the Ion Torch design of the Vi, the Scin began work on a plasma-based thruster. They are more compact, weigh less, and produce significantly more thrust. However, since Ion Torches expel some of their heat in their thrust, the more stable and powerful SC thrusters tend to run hotter.`
+
+
+outfit "SC-12 Plasma Steering"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 95800
+	thumbnail "outfit/small plasma steering scin"
+	"mass" 19
+	"outfit space" -19
+	"engine capacity" -19
+	"turn" 270
+	"turning energy" 0.55
+	"turning heat" 1.8
+	"steering flare sprite" "effect/scin flare/small"
+		"frame rate" 0.5
+	"steering flare sound" "scplasma small"
+	description `The new SC plasma engines allow Gegno Scin ships to outmaneuver their opposing brothers. What might take a Vi ship a long time takes only a moment's worth on a Scin ship.`
+
+
+outfit "SC-14 Plasma Thruster"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 162000
+	thumbnail "outfit/medium plasma steering scin"
+	"mass" 45
+	"outfit space" -45
+	"engine capacity" -45
+	"thrust" 26.8
+	"thrusting energy" 2.8
+	"thrusting heat" 7.9
+	"flare sprite" "effect/scin flare/med"
+		"frame rate" 0.4
+	"flare sound" "scplasma medium"
+	description `SC-14 class thrusters power the Scin's average gunboat or transport, giving their ships reliable speed. They can quickly propel any medium-sized ship forward with reasonable efficiency.`
+
+
+outfit "SC-14 Plasma Steering"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 156400
+	thumbnail "outfit/large plasma engines scin"
+	"mass" 34
+	"outfit space" -34
+	"engine capacity" -34
+	"turn" 690
+	"turning energy" 1.5
+	"turning heat" 4.78
+	"steering flare sprite" "effect/scin flare/med"
+		"frame rate" 0.4
+	"steering flare sound" "scplasma medium"
+	description `Larger than SC-12 engines, SC-14-class steering is one of the most powerful steering systems in Gegno space, allowing the Scin to reposition even their largest ships many times faster than their Vi counterparts.`
+
+
+outfit "SC-15 Plasma Thrusters"
+	plural "SC-15 Plasma Thrusters"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+		"Scin Architect"
+	cost 2312000
+	thumbnail "outfit/medium plasma steering scin"
```

Comment:
The outfit name says "thruster" and the thumbnail here says "steering." Is this correct?

Reply:
Good catches. Got mixed up.